pkg/utils/net-attach-def_test.go

```diff
@@ -280,6 +280,70 @@ var _ = Describe("Netwok Attachment Definition manipulations", func() {
 		})
 	})
 
+	Context("create network statuses from CNI result with multiple interfaces when the IP Interface index isn't set", func() {
+		When("one sandbox interface is specified", func() {
+			It("assigns the IPs to the last sandbox interface specified", func() {
+				cniResult := &cni100.Result{
+					CNIVersion: "1.0.0",
+					Interfaces: []*cni100.Interface{
+						{
+							Name: "foo",
+						},
+						{
+							Name:    "eth0",
+							Mac:     "00:AA:BB:CC:DD:01",
+							Sandbox: "/path/to/network/namespace",
+						},
```

Comment:
could you switch the order of these ?... so we know it grabs the last sandbox, not the last interface, which happens to be the sandbox. 

```suggestion
						{
							Name:    "eth0",
							Mac:     "00:AA:BB:CC:DD:01",
							Sandbox: "/path/to/network/namespace",
						},
						{
							Name: "foo",
						},
```